gammapy/spectrum/results.py

```diff
@@ -0,0 +1,535 @@
+                      e_power = 0, fit_kwargs=None, point_kwargs=None):
+        """Plot full spectrum including flux points and residuals
+
+        Parameters
+        ----------
+        ax : `~matplolib.axes`, optional
+            Axis
+        energy_unit : str, `~astropy.units.Unit`, optional
+            Unit of the energy axis
+        flux_unit : str, `~astropy.units.Unit`, optional
+            Unit of the flux axis
+        e_power : int
+            Power of energy to multiply flux axis with
+        fit_kwargs : dict, optional
+            forwarded to :func:`gammapy.spectrum.results.SpectrumFitResult.plot_fit_function`
+        point_kwargs : dict, optional
+            forwarded to :func:`gammapy.spectrum.results.SpectrumFitResult.plot_points`
+
+        Returns
+        -------
+        ax : `~matplolib.axes`, optional
+            Axis
+        """
+        from matplotlib import gridspec
+        import matplotlib.pyplot as plt
+
+        gs = gridspec.GridSpec(4, 1)
+
+        ax0 = plt.subplot(gs[:-1,:])
+        ax1 = plt.subplot(gs[3,:], sharex=ax0)
+
+        gs.update(hspace=0)
+        plt.setp(ax0.get_xticklabels(), visible=False)
+
+        ax0.set_yscale('log')
+        ax0.set_xscale('log')
+        ax1.set_xscale('log')
+
+        if fit_kwargs is None:
+            fit_kwargs = dict(label='Best Fit {}'.format(self.spectral_model),
+                              barsabove=True, capsize=0, alpha=0.4,
+                              color='cornflowerblue', ecolor='cornflowerblue')
+        if point_kwargs is None:
+            point_kwargs = dict(color='navy')
+
+        self.plot_fit_function(energy_unit=energy_unit, flux_unit=flux_unit,
+                               e_power=e_power, ax=ax0, **fit_kwargs)
+        self.plot_flux_points(energy_unit=energy_unit, flux_unit=flux_unit,
+                              e_power=e_power, ax=ax0, **point_kwargs)
+        self.plot_residuals(energy_unit=energy_unit, ax=ax1, **point_kwargs)
+
+        plt.xlim(self.energy_range[0].to(energy_unit).value * 0.9,
+                 self.energy_range[1].to(energy_unit).value * 1.1)
+
+        ax0.legend(numpoints=1)
+        return gs
+
+
+class SpectrumFitResultDict(dict):
+    """Dict of several `~gammapy.spectrum.SpectrumFitResults`"""
+
+    def info(self):
+        pass
+
+    def to_table(self):
+        """Create overview `~astropy.table.QTable`"""
+
+        val = self.keys()
+        analyses = Column(val, name='Analysis')
+        table = self[val.pop(0)].to_table()
+        # Add other analysis with correct units
+        for key in val:
+            temp = QTable(self[key].to_table())
+            new_row = list()
+            for col in table.columns:
+                new_row.append(temp[col].to(table[col].unit).value)
+            table.add_row(new_row)
+
+        table.add_column(analyses, index=0)
+        return table
+
+    def overplot(self):
+        """Overplot spectra"""
+        pass
```

Comment:
I usually put `raise NotImplementedError` for functions / methods that aren't implemented yet.
If you put `pass` the user gets no feedback that something is wrong.